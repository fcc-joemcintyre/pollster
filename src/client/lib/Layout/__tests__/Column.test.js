/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { Column } from '..';

test ('Column renders default component', () => {
  const component = renderer.create (
    <Column />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});
