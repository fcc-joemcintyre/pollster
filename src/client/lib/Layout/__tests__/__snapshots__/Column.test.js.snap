// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column renders default component 1`] = `
.c0 {
  width: 100%;
  display: inline-block;
  vertical-align: top;
  margin-right: 20px;
}

.c0:last-child {
  margin-right: 0;
}

<div
  className="c0"
/>
`;
